config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
@tmatsuo indeed it does, but I don't consider DEBUG=off or similar a good practice.

> It is clearly better than other options.

Clearly I disagree =).